chart/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: nexus-iq-server-ha
-version: 171.0.0
+version: 171.1.0
```

Comment:
Shouldn't need to change the version so I'd make sure this is up-to-date with main